packages/core/core/src/services/document-service/internationalization.ts

```diff
@@ -70,7 +55,7 @@ const localeToData: Transform = (contentType, params) => {
     return params;
   }
 
-  if (params.locale) {
+  if (params.locale && typeof params.locale === 'string') {
```

Comment:
what happens if locale is * here ?

Reply:
I think you're right and we shouldn't allow that. This is only used in the document service create
and update but we'd never want to put '*' into the data